packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.types.ts

```diff
@@ -109,7 +109,7 @@ export interface IFocusZoneProps extends React.HTMLAttributes<HTMLElement | Focu
    * an unfortunate side effect is that users will not be able to tab out of the focus zone
    * and have to hit escape or some other key.
    */
-  allowTabKey?: boolean;
```

Comment:
removing this makes this a breaking/major change. Keep it around and set it to be deprecated.